src/bin/gpfdist/gpfdist.c

```diff
@@ -836,6 +843,29 @@ static void parse_command_line(int argc, const char* const argv[],
 			exit(1);
         }
     }
+
+	/* validate opt.X */
+	if (opt.X)
+	{
+		char* p;
+		size_t p_len;
+
+		if (!opt.trlist)
+			usage_error("Error: default transformation is set,"
+				" but TRANSFORMATIONS are not specified", 0);
+
+		p = gstring_trim(apr_pstrdup(pool, opt.X));
+		p_len = strlen(p);
```

Comment:
For other options the value allocated in apr_pstrdup (variable `p`) is assigned to `opt`, for ex:

```
		opt.d = p;
```
and 
```
		opt.l = p;
```
and
```
		opt.ssl = p;
```

why the `p` is not assigned to `opt.X`?

Reply:
That was a mistake. Thanks.